### PR TITLE
The dynamic change of path property throws an exception in Azure file provider

### PR DESCRIPTION
**Pull request by kmkrish001**
The dynamic change of path property throws an exception in the Azure file provider for the File Manager component.

### CODE REVIEW

**Comment by kmkrish001** [comment]
https://github.com/essential-studio/azure-aspcore-file-provider/pull/15

**Review by thirukumaran-murugan — APPROVED** [review]
Hi @gsumankumar ,

The changes are fine, could you please merge the PR.

Regards,
Thirukumaran M.

File: Models/AzureFileProvider.cs [hunk 1]
```diff
@@ -64,6 +64,16 @@ protected async Task<FileManagerResponse> GetFilesAsync(string path, string filt
             FileManagerDirectoryContent cwd = new FileManagerDirectoryContent();
             try
             {
+                var blobPages = container.GetBlobsAsync(prefix: path).AsPages().GetAsyncEnumerator();
+                await blobPages.MoveNextAsync();
+                bool directoryExists = blobPages.Current.Values.Count() > 0;
+                if (!directoryExists) {
+                    ErrorDetails er = new ErrorDetails();
+                    er.Message = "Could not find a part of the path " + "'" + path + "'" + ".";
```
Comment (gsumankumar) [inline Models/AzureFileProvider.cs:72]: do we have localization on error message?

Comment (thirukumaran-murugan) [reply Models/AzureFileProvider.cs:72]: Hi @gsumankumar ,

We don't have location support for this. It's common file not fount related exception we throw in the component level.

Comment (gsumankumar) [reply Models/AzureFileProvider.cs:72]: Customer can modify this code based on their need.

**Review by gsumankumar — APPROVED** [review]

**Pull request merged**